Makefile

```diff
@@ -488,7 +489,15 @@ uninstall:
 PYTHON_SCRIPTS := $(shell grep 'python3' -n utils/* 2>/dev/null | grep ':1:' | sed 's/^\([^:]*\):.*$$/\1/g')
 
 .PHONY: check
-check: check-cfgen check-hax check-miniprov flake8 mypy
+check: check-cfgen check-hax check-miniprov flake8 mypy gcov
+
+.PHONY: gcov
+gcov:
+	@$(call _info, Checking gcov package.)
+	@if [ -z $(COV_BIN) ]; then \
+	$(call _log, installing gcov); \
+		yum install -y lcov; \
```

Comment:
I don't think it is a good practice. Makefile CAN suggest which packages are needed but it should not install them. The user may install gcov in some other ways than running `yum` (`make install`? snap? flatpak? nix?)

Reply:
Added log message. Removed installation part.